### PR TITLE
ABP change the arg name in the comment and update req 

### LINKED ISSUE #454: [BUG]: Improvement-correcting the ABP example comment in the training script and req
### Version

22.11

### Which installation method(s) does this occur on?

_No response_

### Describe the bug.

there is an extra hyphen in the comment that needs to be removed. The comment shows how the script should be run.

### Minimum reproducible example

_No response_

### Relevant log output

_No response_

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by gbatmaz**
in the example python statement, there was an extra hyphen, and autogenerated req was missing libraries, and adding sklearn
Closes #454

### CODE REVIEW

**Comment by gbatmaz** [comment]
Closes #454

**Comment by gbatmaz** [comment]
The issue was included in the description and in a comment but this check fails
```
Error: Request failed due to following response errors:
 - Resource not accessible by integration
```

**Review by raykallen — APPROVED** [review]
lgtm

**Comment by raykallen** [comment]
@gpucibot merge
